triagebot.toml

```diff
@@ -106,10 +106,16 @@ resolved/implemented on Fuchsia. Could one of you weigh in?
 """
 label = "O-fuchsia"
 
-[ping.macos]
+[ping.apple]
+alias = ["macos", "ios", "tvos", "watchos", "visionos"]
 message = """\
-Hey MacOS Group! This issue or PR could use some MacOS-specific guidance. Could one
-of you weigh in? Thanks <3
+Hey Apple notification group! This issue or PR could use some Apple-specific
+guidance. Could one of you weigh in? Thanks <3
+
+(In case it's useful, here are some [instructions] for tackling these sorts of
+issues).
+
+[instructions]: https://rustc-dev-guide.rust-lang.org/notification-groups/apple.html
 """
 label = "O-macos"
```

Comment:
Unsure here what label we should use, or if it makes sense to create a new sort of super-label for all of the Apple platforms?

Reply:
I created https://github.com/rust-lang/rust/labels/O-apple.

I'm approving this PR as is to unbreak "rustbot ping apple" (https://github.com/rust-lang/rust/pull/123723#issuecomment-2094894269). If you get to this before bors merged the PR, feel free to update this PR and use r=dtolnay, otherwise follow up with a new PR to change this label in triagebot.toml (or let me know if you would prefer some different name for the super-label).

Reply:
I'm fine with O-apple

r=dtolnay